graphql/resolvers.go

```diff
@@ -1402,6 +1403,26 @@ func (r *queryResolver) SiteBanner(ctx context.Context) (*restModel.APIBanner, e
 	return &banner, nil
 }
 
+func (r *queryResolver) SpruceConfig(ctx context.Context) (*restModel.APIAdminSettings, error) {
+	config, err := evergreen.GetConfig()
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error Fetching evergreen settings: %s", err.Error()))
+	}
+
+	spruceConfig := restModel.APIAdminSettings{}
+	err = spruceConfig.BuildFromService(config)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error building api admin settings from service: %s", err.Error()))
+	}
+	uiConfig := restModel.APIUIConfig{}
+	err = uiConfig.BuildFromService(config.Ui)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error building api UI Config from service: %s", err.Error()))
+	}
+	spruceConfig.Ui = &uiConfig
```

Comment:
it is also possible to do this mapping in APIAdminSettings.BuildFromService. I'm not sure if doing that would have an additional benefit so I think it's fine here